portal_multicurrency_totals/demo/res_partner_demo.xml

```diff
@@ -2,24 +2,21 @@
 <odoo>
 
     <!-- Grant portal access to the customer -->
-    <record id="grant_portal_access" model="portal.wizard">
+    <record id="grant_portal_access" model="portal.wizard.user">
         <field
-            name="user_ids"
+            name="user_id"
             model="res.partner"
             eval="[
             (0, 0, {
```

Comment:
Migrate to the command syntax.

And let's the chance to fix indentation here.

Reply:
Fixed.